src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Address.java

```diff
@@ -10,16 +10,25 @@
 import lombok.Builder;
 import lombok.Data;
 import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.sscs.ccd.validation.address.Postcode;
+import uk.gov.hmcts.reform.sscs.ccd.validation.groups.UniversalCreditValidationGroup;
+import uk.gov.hmcts.reform.sscs.ccd.validation.string.StringNoSpecialCharacters;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @Data
 @Builder(toBuilder = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class Address {
+
+    @StringNoSpecialCharacters(fieldName = "Line 1", groups = {UniversalCreditValidationGroup.class})
```

Comment:
This is used by other benefit types already - will it work for existing functionality?

Reply:
Yes, it will.   I've been careful to ensure that nothing we've added here affects anything else using Address.   
These annotations only kick in in Address is validated explicitly, eg

``` 
validator.validate(address); 

```
 or if Address is annotated with @Valid on all parent objects, eg, for the Appellant address,  we'd have to add @Valid to the appeal of SscsCaseData.

```
    @Valid
    private Appeal appeal;

```
@Valid to the Appellant within appeal

```
    @Valid
    private Appellant appellant;
```
and @Valid to the address of Appellant

```
  @Valid
    private Address address;
```

Neither of these conditions hold -  I don't believe we validate Address explicilty elsewhere in the codebase, and I don't believe don't have @Valid annotations associated with Address attributes anywhere.

However, just in case we did validate Address anywere else, either now or in the future, I've added an additional requirement, that for these validations to take place,  the validation context has to be for the UniversalCreditValidationGroup.     

This group is inactive by default,  and enabled explicitly on the UC-specific address attribute of SscsCaseData

```
    @Valid
    @ConvertGroup(to = UniversalCreditValidationGroup.class)
    private Address jointPartyAddress;
```

I've added tests to ensure that this validation does not affect existing addresses in the tests below:

JointPartyAddressAndAppellantAddressValidationTest
NonUniversalCreditAddressCanContainSpecialCharactersTest
NonUniversalCreditAddressPostcodeCanBeInvalidTest

If we ever wanted to change this behaviour in future, and share this validation across the different uses of Address, we can change the group conditions as appropriate.

I'll add a comment to this effect the the tests as per your other comment.